### PR TITLE
feat(web): Added a Ukraine chat panel to an article (#7022)

### PR DESCRIPTION
**Pull request by RunarVestmann**
Cherry-pick merge commit from PR https://github.com/island-is/island.is/pull/7022 to release branch. See the PR for more details on the changes.

### CODE REVIEW

**Review by rafnarnason — APPROVED** [review]
lgtm

**Review by pshomov — APPROVED** [review]
LGTM

File: apps/web/screens/Article/Article.tsx [hunk 1]
```diff
@@ -283,6 +284,52 @@ const ArticleSidebar: FC<ArticleSidebarProps> = ({
   )
 }
 
+const activateUkraineChatPanel = () => {
```
Comment (kristofer-lp8f) [inline apps/web/screens/Article/Article.tsx:287]: Is this really Ukraine specific or just chat panel?

File: apps/web/screens/Article/Article.tsx [hunk 2]
```diff
@@ -338,6 +385,16 @@ const ArticleScreen: Screen<ArticleProps> = ({
     150,
   )
 
+  useEffect(() => {
+    let widget: any
+    if (article.id === '7i92Z9s9HQeYlpGReYQVX') {
```
Comment (kristofer-lp8f) [inline apps/web/screens/Article/Article.tsx:390]: Hard-coded I'd 🤔

**Review by kristofer-lp8f — APPROVED** [review]
Not sure about hard-coding, but otherwise LGTM 👍

**Pull request merged**